src/components/Overview/__tests__/Overview.test.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Overview } from '../index';
+import { getChildPageNodes } from '@/utils/getChildPageNodes';
+
+const routerMock = {
+  __esModule: true,
+  useRouter: () => {
+    return {
+      query: { platform: 'react' }
+    };
+  }
+};
+
+jest.mock('next/router', () => routerMock);
+
+describe('Overview', () => {
+  const childPageNodes = getChildPageNodes('/[platform]/build-a-backend');
+  const component = <Overview childPageNodes={childPageNodes} />;
+
+  it('should render the Overview component', async () => {
+    render(component);
+    const overviewNode = await screen.findAllByRole('link');
```

Comment:
do you need findAllByRole if you are just using the first one?

Reply:
Yes because there aren't props to differentiate and be able to grab one.